onnx/test/cpp/onnxifi_backend_test.cc

```diff
@@ -55,8 +55,10 @@ namespace ONNX_NAMESPACE
 			EXPECT_EQ_OSS(dummy_backend.onnxGetBackendCompatibility(backendID, onnxModelSize, onnxModel));
 
 			//Testing onnxInitBackend
-			uint64_t auxpropertiesList = 0;
-			EXPECT_EQ_OSS(dummy_backend.onnxInitBackend(backendID, &auxpropertiesList, &backend));
+			const uint64_t backendProperties[] = {
+                            ONNXIFI_BACKEND_PROPERTY_NONE
+                        };
```

Comment:
indent?

Reply:
I decided to not reformat the whole file in this PR, will rebase & reformat after #1259 lands